person.rb

```diff
@@ -16,6 +16,10 @@ def initialize(name = 'Unknown', age = 0, parent_permission: true)
     super()
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association you can do:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
- [ ] This method `rental` has to be available in the `book` class as well and remember the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. you can do:
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
> 

i should use this in books.rb?

Reply:
@DeliceLydia hello